planning/planning-so-i-do-not-fail.jpeg.md

```diff
@@ -0,0 +1,86 @@
+# Plan
```

Comment:
Very detailed yet succinct plan, definitely one that will not cause you to fail!

Reply:
Thanks, have to admit having a checkbox to tick off was nice to have. Maybe I should take some effort planning these things...